src/sockperf.cpp

```diff
@@ -2093,7 +2096,7 @@ void cleanup() {
     }
 #ifdef USING_VMA_EXTRA_API
     if (g_vma_api && s_user_params.is_vmazcopyread) {
-        for (int i = s_fd_min; i < s_fd_max; i++) {
+        for (int i = s_fd_min; i <= s_fd_max_zcopy; i++) {
```

Comment:
You can do it w/o `for` cycle. See https://www.geeksforgeeks.org/different-ways-delete-elements-stdmap-erase-clear/